sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseService.java

```diff
@@ -1313,4 +1313,55 @@ public boolean isCaseEditAllowed(Case caze) {
 	public void updateExternalData(List<ExternalDataDto> externalData) throws ExternalDataUpdateException {
 		ExternalDataUtil.updateExternalData(externalData, this::getByUuids, this::ensurePersisted);
 	}
+
+	public void updateCompleteness(String caseUuid) {
+		Case caze = getByUuid(caseUuid);
+		caze.setCompleteness(calculateCompleteness(caze));
+		ensurePersisted(caze);
+	}
+
+	@TransactionAttribute(TransactionAttributeType.REQUIRES_NEW)
+	public void updateCompleteness(List<String> caseUuids) {
+		List<Case> casesForUpdate = getByUuids(caseUuids);
```

Comment:
Nope, reuse `updateCompleteness(String caseUuid)` to avoid code duplication

Reply:
updated